Add test coverage for --exclude argument

sonro opened the pull request:
Fixes #127

Once this pull request is merged, I will open an additional pull request with a refactor of the code in [cli_test.rs](https://github.com/mgrachev/dotenv-linter/blob/master/tests/cli_test.rs); as there is now excessive code repeition in the tests.

<!--
Thank you for your pull request. Please provide a description above and review
the requirements below.

Bug fixes and new features should include tests.
-->

<!-- _Please make sure to review and check all of these items:_ -->

#### ✔ Checklist:
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] This PR has been added to [CHANGELOG.md](https://github.com/mgrachev/dotenv-linter/blob/master/CHANGELOG.md) (at the top of the list);

<!-- _NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open._ -->

mgrachev commented:
@sonro 👋Thank you for your contribution! 🚀

What do you think about moving tests for `--exclude` arguments to another file?

sonro commented:
@mgrachev I have refactored the tests to smaller files and added helper functions too. Was going to put that in another pull request as it is a bigger change. Can push it all here if you would like?

mgrachev commented:
@sonro Thank you! Another PR would be better.
